package.json

```diff
@@ -12,7 +12,7 @@
     "url": "http://github.com/ghost-inspector/node-ghost-inspector.git"
   },
   "engines": {
-    "node": ">=8.9.0"
+    "node": ">=10.18.0"
```

Comment:
Does this need to specifically be 10.18? or could it just be 10?

Reply:
I don't think so, that's just the latest LTS I think. I've adjusted this to `10.0`.